### DISCUSSION TITLE
Show context feature for loki data source explorer when using `label filter expression`

### DISCUSSION THREAD

**Discussion by jbreiding** [grafana/grafana#42754] [General]
Issue created in response to the behavior seen.
https://github.com/grafana/grafana/issues/41448

The show context feature of the log explorer is really powerful, using the label filter expression also helps in cases where the logs are structured and can be filtered to specific fields.

Having this feature available for both `line filter expressions` and `label filter expressions` would be very helpful and powerful.

**Comment by hunkjun** [comment]
I hope so too. Like this https://github.com/grafana/grafana/issues/50913.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
